pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -848,26 +869,18 @@ func (t *Test) choosePreviousReleases() ([]*clusterupgrade.Version, error) {
 			return nil, err
 		}
 
-		if predPred.AtLeast(minSupportedSkipVersionUpgrade) {
-			// If the predecessor's predecessor supports skip-version
-			// upgrades and we haven't performed a skip-version upgrade yet,
-			// do it. This logic makes sure that, when skip-version upgrades
-			// are enabled, it happens when upgrading to the current
-			// release, which is the most important upgrade to be tested on
-			// any release branch.
-			if numSkips == 0 {
-				numSkips++
-				return []*clusterupgrade.Version{predPred}, nil
-			}
-
-			// If we already performed a skip-version upgrade on this test
-			// plan, we can choose to do another one or not.
-			return []*clusterupgrade.Version{pred, predPred}, nil
+		// If we haven't performed a skip-version upgrade yet, do it. This logic
+		// makes sure that, when skip-version upgrades are enabled, it happens
+		// when upgrading to the current release, which is the most important
```

Comment:
> it happens when upgrading to the current release

I think this will also make it so if skip-versions aren't supported to the current release, it will always attempt a skip-version upgrade for the predecessor. I think that's arguably desirable behavior though? (just trying to reason that it's fine)

Reply:
Yeah. I think it's fine because we also have the `skipUpgrades` randomization, so in some runs we won't skip anything.